gp-templates/project.php

```diff
@@ -48,12 +49,16 @@ function( $items ) {
 </div>
 <?php endif; ?>
 
-<div id="project" <?php if ( $sub_projects ) {
-	echo ' class="with-sub-projects"'; } ?>>
+<div id="project" 
+<?php
+if ( $sub_projects ) {
+	echo ' class="with-sub-projects"'; }
+	?>
+	>
```

Comment:
This will cause a lot of unnecessary whitespace in the HTML markup. The following is the preferred approach.

```php
<?php
$class = $sub_projects ? ' class="with-sub-projects"' : ''; 
?>
<div id="project"<?php echo $class; ?>>

Reply:
Fixed